### PR TITLE
Add Okta support

### PR DESCRIPTION
**Pull request by dimaryaz**

### CODE REVIEW

**Comment by akarve** [comment]
@nl0 can you drop an Okta log into the button (maybe in a PR against this one) and make the button widths centered and identical for all SOO providers?

**Comment by nl0** [comment]
> @nl0 can you drop an Okta log into the button (maybe in a PR against this one) and make the button widths centered and identical for all SSO providers?

@akarve done in #1735 

which option do you guys like best?

1
sso-btns-align-1

2
sso-btns-align-2

3
sso-btns-align-3

**Comment by akarve** [comment]
@nl0 let's do style 3, we want the icons all aligned

File: catalog/app/containers/Auth/SSOOkta.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  const intl = useIntl()
+  const { push: notify } = Notifications.use()
+  const { urls } = NamedRoutes.use()
+
+  const handleClick = React.useCallback(() => {
+    if (mutex.current) return
+    mutex.claim(MUTEX_POPUP)
+
+    const oktaDomain = `https://${cfg.oktaCompanyName}.okta.com`
+    const nonce = '1' // TODO
```
Comment (akarve) [inline catalog/app/containers/Auth/SSOOkta.js:40]: Let's do `hash(email + epochtime)`

Comment (dimaryaz) [reply catalog/app/containers/Auth/SSOOkta.js]: Oops... forgot to `git push`. Generating a random number now.

**Pull request merged**